src/libstd/lib.rs

```diff
@@ -352,18 +353,22 @@ use prelude::v1::*;
                  debug_assert_ne, unreachable, unimplemented, write, writeln, try)]
 extern crate core as __core;
 
-#[allow(deprecated)] extern crate rand as core_rand;
+#[doc(masked)]
+#[allow(deprecated)]
+extern crate rand as core_rand;
 #[macro_use]
 #[macro_reexport(vec, format)]
 extern crate alloc;
 extern crate alloc_system;
 extern crate std_unicode;
+#[doc(masked)]
 extern crate libc;
 
 // We always need an unwinder currently for backtraces
 extern crate unwind;
```

Comment:
`unwind` would also need to be `doc(masked)` to get rid of the impls for `_Unwind_Reason_Code` and `_Unwind_Action` on the [`Clone`](https://doc.rust-lang.org/nightly/std/clone/trait.Clone.html#implementors) page for example.